app/server/app/routes/data.js

```diff
@@ -1,11 +1,153 @@
-const { resolve } = require("node:path");
-const { readFile } = require("node:fs/promises");
-const express = require("express");
 const axios = require("axios");
+const express = require("express");
+const { resolve } = require("node:path");
+const Papa = require("papaparse");
+const pg = require("pg");
+const QueryStream = require("pg-query-stream");
+const Excel = require("exceljs");
+const { getActiveSchema } = require("../middleware");
+const { knex } = require("../utilities/database");
```

Comment:
Looks like some of these aren't being used anymore:
```suggestion
const express = require("express");
const Excel = require("exceljs");
const { getActiveSchema } = require("../middleware");
const { knex } = require("../utilities/database");
```